### ISSUE TITLE
Update all package typings

### ISSUE THREAD

**Issue by huan086** [huan086/lingui-typings#3]
The macro function returns objects rather than string. The i18n._ function now takes an object for the id parameter

**Comment by honi** [comment]
Hello, is there any plans on updating the typings for Lingui v3?

And thank you very much for the typings!

**Comment by huan086** [comment]
No. The lingui library itself should have typings out of the box in v3

**Comment by honi** [comment]
Awesome, I've just configured v3.0.0-13 and can confirm it's working out of the box with TypeScript!
